system-addon/lib/HighlightsFeed.jsm

```diff
@@ -118,6 +118,8 @@ this.HighlightsFeed = class HighlightsFeed {
 
     SectionsManager.updateSection(SECTION_ID, {rows: this.highlights}, this.highlightsLastUpdated === 0 || broadcast);
     this.highlightsLastUpdated = Date.now();
+    const updatedAction = {type: at.HIGHLIGHTS_UPDATED};
+    this.store.dispatch(ac.SendToMain(updatedAction));
```

Comment:
@rlr I'm not sure about this approach. The SectionManager already has a system in place for notifying when a section is updated. Doing this would bypass it. I would get a second opinion on this.